core-it-suite/src/test/resources/mng-4403/repo/org/apache/maven/its/mng4403/b/0.1/b-0.1.pom

```diff
@@ -80,8 +80,7 @@ under the License.
       <!-- duplicate dependency with different version (MNG-4005), last declaration wins -->
       <groupId>org.apache.maven.its.mng4403</groupId>
       <artifactId>a</artifactId>
-      <!-- deprecated expression prefix (MNG-4421) -->
-      <version>${pom.version}</version>
+      <version>${project.version}</version>
```

Comment:
Attention, if this file has a chksum it must also be updated

Reply:
Good catch.